assets/components/src/checkbox-control/style.scss

```diff
@@ -1,78 +0,0 @@
-/**
- * Checkbox styles.
- */
-
-@import '../../../shared/scss/colors';
-@import '~@wordpress/base-styles/colors';
-
-.newspack-checkbox-control {
-	color: $gray-700;
-	display: flex;
-	font-size: 14px;
-	line-height: 24px;
-	margin: 32px 0;
```

Comment:
Maybe outside the scope of this PR, but removing the margins causes some Settings components to look strange. For example, in the Reader Revenue > Stripe Settings page:

Before:

<img width="1052" alt="Screen Shot 2022-06-09 at 1 26 08 PM" src="https://user-images.githubusercontent.com/2230142/172928840-60078e23-99ba-4f44-9d10-8d32c4b6e3a1.png">

After:

<img width="1048" alt="Screen Shot 2022-06-09 at 1 25 29 PM" src="https://user-images.githubusercontent.com/2230142/172928859-0fb1e6bc-8411-404a-9d55-c31ba23b3b55.png">

Reply:
I've fixed the Stripe layout here: https://github.com/Automattic/newspack-plugin/pull/1676/commits/99b4ea41a3bd61f61dbd4a8e3ce7ad834240bb41

Did you notice more broken Settings components?

Reply:
I noticed the Campaigns Settings were also broken: https://github.com/Automattic/newspack-plugin/pull/1676/commits/89919197c2434145e8b003ee34d179fe0d609aa5

Reply:
Didn't notice any other broken settings components. Campaigns does look better now (the spinner positioning I highlighted in another comment came from Campaigns settings).

Reply:
Thank you @dkoo.